api/calculate_frequencies.py

```diff
@@ -28,50 +28,21 @@
 """
 def calculate_frequencies3(genotypedata, alleles_definitions):
 
```

Comment:
Move the method command above inside the method, as a docstring (per standard python format); minor formatting things, but for the comment use single quotes instead of double quotes, and use the `:param (...): (...)`/`:return: (...)` format used by the other methods instead of input/output as it currently uses (just so we're using consistent comment formats across the project)

Reply:
Also, could you elaborate a little bit on what "calculate frequencies of alleles" means?